extensions/mongoProcessor/MongoQueueProcessor.js

```diff
@@ -28,6 +28,9 @@ const { metricsExtension, metricsTypeCompleted } =
 // TODO - ADD PREFIX BASED ON SOURCE
 // april 6, 2018
 
+// Copied from cloudserver "emptyFileMd5"
+const ZERO_BYTE_ETAG = 'd41d8cd98f00b204e9800998ecf8427e';
```

Comment:
Looks like this is also available from Arsenal as a constant `emptyFileMd5`, and could be imported here. 😸